analyzer/evmnfts/evm_nfts.go

```diff
@@ -118,6 +118,7 @@ func (p *processor) ProcessItem(ctx context.Context, batch *storage.QueryBatch,
 		staleNFT.DownloadRound,
 		nftData.MetadataURI,
 		nftData.MetadataAccessed,
+		nftData.Metadata,
```

Comment:
apparently this makes it so that the string is the JSON serialized value. I have no idea how, hypothetically, to tell the db  that "the JSON value was a string, here's that string."